### PR TITLE
Fix validation-group not working properly

### LINKED ISSUE #210: Validation Group and form checkValid
Hello,

It looks like checkValid for a form returns false unless everything in the validation group is valid.  You can see this in your example by only selecting 1 checkbox https://github.com/huei90/angular-validation/blob/master/index.html#L325

Shouldn't the form be valid if at least one is selected?

### PR DESCRIPTION
**Pull request by lvarayut**
This PR fixes the validation-group problem, the validation-group didn't properly set validity of other elements in a group. 

Closes #210

### CODE REVIEW

**Comment by lvarayut** [comment]
@huei90, will merge after your confirmation.

File: src/validator.directive.js [hunk 1]
```diff
@@ -89,29 +90,35 @@
...
+    };
+
+    /**
+     * Set validity to all elements inside the given group
+     *
+     * @param scope
+     * @param groupName
+     * @param validity
+     */
+    function setValiationGroup(scope, validationGroup, validity) {
```
Comment (huei90) [inline src/validator.directive.js]: type: setValidationGroup

**Comment by huei90** [comment]
This PR contains many changes but I believe on you 😄 Feel free to merge it.

**Comment by huei90** [comment]
@Nazanin1369

**Comment by huei90** [comment]
btw, great job 🍻

File: src/validator.directive.js (hunk 1 above)
Comment (lvarayut) [inline src/validator.directive.js]: That's kind of embarrassing...., fixing

**Comment by lvarayut** [comment]
I wrote more tests on it. So, it should be reliable enough. 😄

**Pull request merged**